gridt/src/app/home/expandable/expandable.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input } from "@angular/core";
+
+@Component({
+  selector: "app-expandable",
+  templateUrl: "./expandable.component.html",
+  styleUrls: ["./expandable.component.scss"],
+})
+export class ExpandableComponent implements OnInit {
+
+  @Input() signalMessage;
+  itemExpanded = false;
+
+  constructor() { }
+
+  ngOnInit() {}
```

Comment:
You can leave these two functions as well as the implementation of ngOnInit entirely.